pupa/scrape/event.py

```diff
@@ -23,7 +24,7 @@ def add_vote_event(self, vote_event, *, id=None, note='consideration'):
         self.add_entity(name=vote_event, entity_type='vote_event', id=id, note=note)
 
     def add_committee(self, committee, *, id=None, note='participant'):
-        self.add_entity(name=committee, entity_type='committee', id=id, note=note)
+        self.add_entity(name=committee, entity_type='organization', id=id, note=note)
```

Comment:
Docs will need updating here for `add_participant` type:
http://opencivicdata.readthedocs.org/en/latest/scrape/events.html